mods/ra/sequences/structures.yaml

```diff
@@ -93,7 +96,7 @@ proc:
 		Length: *
 	dead: procdead
 		Start: 0
-		ZOffset: -1c511
```

Comment:
I was right, removing this makes the death animation display on top of (harvester) husks.